csharp/ql/test/resources/stubs/_frameworks/Microsoft.AspNetCore.App/Microsoft.AspNetCore.Razor.Runtime.cs

```diff
@@ -22,6 +22,7 @@ public abstract class RazorCompiledItem
                     public abstract System.Collections.Generic.IReadOnlyList<object> Metadata { get; }
                     public abstract System.Type Type { get; }
                 }
+                [System.AttributeUsage(System.AttributeTargets.Assembly | System.AttributeTargets.Class, AllowMultiple=true, Inherited=false)]
```

Comment:
Now that https://github.com/github/codeql/pull/14609 has been merged, you can revert this change and rebase onto latest `main` instead.